### DISCUSSION TITLE
Tokens of gratitude

### DISCUSSION THREAD

**Discussion by slackhideo** [Aelto/W3_RandomEncounters_Tweaks#150] [Q&A] [answered]
First, I want to thank you for the mega job you are doing with this mod. It's great!
I have a question about exchanging tokens of gratitude for rewards. I can see what type of rewards I can get from noticeboard contracts (mostly consumables at the beginning of the game, apparently) but when I actually do the exchange I cannot see what I'm earning. Is this the way it works? Or maybe I have an issue on my end?
If this is the way it is programmed, maybe you could add some message to let us know our rewards, please?

**Comment by Aelto** [comment, marked as answer]
Hello! I'm glad you're enjoying the mod thank you!

I think the combat/item log displays the items you get from trading tokens but i'm sure, like me, you have this element of the HUD disabled. This wouldn't be too complicated to do, i'll add a ticket and see if i can squeeze it in the next release i have planned!

**Reply by slackhideo** [reply]
Thank you for your swift reply!
If the combat/item log you mentioned is that one on the bottom left corner that displays "Received: Item Name" when you loot something, then it is not showing what we get from exchanging tokens of gratitude. I have it enabled, actually, and I just tested it with two tokens of gratitude.
What I saw happening was that the quick access items (keys R, F, T, and Y), just below the aforementioned log, briefly disappeared in the exact moment I did the exchange. Also, I got one gear item and it played the sword sound. But no sign of the item name appearing on the screen.

**Reply by Aelto** [reply]
There are cases where a token may give no items at all too, depending on your settings and the token you give the chances aren't always 100% to get a successful roll with at least 1 item. But before you tell it is not great you should know you can also get multiple items in a single roll 🤗 That could be the reason you didn't get any message from the trades too?

But anyway, i'll try to add some sort of log for the future releases! Thanks!

**Reply by slackhideo** [reply]
I see. I'll do some more investigation on my end.
Thank you very much for your time and attention! I'm looking forward to seeing your future releases 😊

**Comment by binmao** [comment]
I have got the message ,thank you!     您发给我的信件已经收到。
